.github/actions/build-and-push/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build and Push
+description: Build and Push docker image to dibbs-query-connector registry
+
+inputs:
+  username:
+    description: GitHub username of the user who triggered the workflow
+    required: true
+  password:
+    description: Used to log in to the Container registry
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Check Out Changes
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Log in to the Container registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ inputs.username }}
+        password: ${{ inputs.password }}
+
+    - name: lowercase repo name to use in following step
+      shell: bash
+      env:
+        REPO: ${{ github.repository }}
+      run: |
+        echo "REPO=${GITHUB_REPOSITORY,,}" >> $GITHUB_ENV
```

Comment:
Question! Any particular reason why we opted to make this into a variable? Theoretically, this should never change. This seems like it's a little overkill.

Reply:
I'm assuming whomever initially created the CD workflow added it so it would align with the value configured in [phdi-charts](https://github.com/CDCgov/phdi-charts/blob/1fd0de9c14a8b480edc5cdf1be9b1d66977a9e86/charts/tefca-viewer/values.yaml#L5). 

I can modify it and create a PR in phdi-charts to change this as well